EasyPost.Tests.Net/CarrierAccountTest.cs

```diff
@@ -8,56 +7,101 @@ namespace EasyPost.Tests.Net
     public class CarrierAccountTest
     {
         [TestInitialize]
-        public void Initialize() => ClientManager.SetCurrent("GxhY479LTioDWsGcEtSAfQ");
+        public void Initialize()
+        {
+            VCR.SetUp(VCRApiKey.Production, "carrier_account", true);
+        }
+
+        private static CarrierAccount CreateBasicCarrierAccount()
+        {
+            return CarrierAccount.Create(Fixture.BasicCarrierAccount);
+        }
 
-        [Ignore]
         [TestMethod]
-        public void TestCRUD()
+        public void TestCreate()
         {
-            CarrierAccount account = CarrierAccount.Create(new Dictionary<string, object>
+            VCR.Replay("create");
+
+            CarrierAccount carrierAccount = CreateBasicCarrierAccount();
+
+            Assert.IsInstanceOfType(carrierAccount, typeof(CarrierAccount));
+            Assert.IsTrue(carrierAccount.id.StartsWith("ca_"));
```

Comment:
Can we also assert the type here per the other libs?

Reply:
We already do on line 27